schedule/yast/sle/guided_xfs/guided_xfs@s390x_kvm.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name:           guided_xfs
+description:    >
+  Guided Partitioning installation with xfs filesystem.
+vars:
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  guided_open:
```

Comment:
we need to simplify for now `schedule/yast/sle/flows/default_s390x_kvm.yaml` offering only one hook `guided_partitioning` which we will override here. There is no need to have some many hooks (I did that for a reason in the first iteration of this work but we can solve it better later). We have done similar in another PR already.